ManageIQ/dead_methods.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#all temp files
+ROUTES=`mktemp`
+ALLOWEDMETHODS=`mktemp`
+OLD=`mktemp`
+NEW=`mktemp`
+
+#in miq get routes
+bundle exec rake routes | ag "\s#*([\S]+)$" -o | ag '(?<=#).*' -o > "$ROUTES"
+echo "routes DONE"
+
+#in miq-ui
+cd ..
+cd manageiq-ui-classic
+echo "moved to miq-ui-classic"
+
+if [[ -n $0 ]]; then
+  git checkout $0
+  bin/update
+fi
+ruby  /Users/zita/Desktop/all.rb > "$ALLOWEDMETHODS"
+sed -i "" '/^\*/ d' "$ALLOWEDMETHODS"
```

Comment:
to me, this always fails with 

```bash
sed: can't read /^\*/ d: No such file or directory
```

I must be doing something wrong

Reply:
Mac vs Linux:

in macs, -i takes a required parameter (which extension to add to the backup file)
on linux, -i takes an optional parameter, and space means it's not there

=> `sed -i" " ...` should work on both